pom.xml

```diff
@@ -48,6 +48,16 @@
             <artifactId>mybatis</artifactId>
             <version>3.5.11</version>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>5.9.0</version>
```

Comment:
Can we use an older version? I'm not confident with using a newer one.

Reply:
I really need this latest version.